spec/setup.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Test setup for server-side tests.
+ */
+var chai = require("chai");
+var sinonChai = require("sinon-chai");
+
+// Add chai plugins.
+chai.use(sinonChai);
+
+// Add test lib globals.
+global.expect = chai.expect;
+global.assert = chai.assert;
```

Comment:
good call - I'm used to setting up tests where there is some frontend and some back end tests so the stack is a bit more involved.